public/index.html

```diff
@@ -0,0 +1,171 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/public/css/reset.css" rel="stylesheet" />
+    <link href="/public/css/style.css" rel="stylesheet" />
+    <link href="/public/css/components/header.css" rel="stylesheet" />
+    <link href="/public/css/components/features.css" rel="stylesheet" />
+    <link href="/public/css/components/feature-list.css" rel="stylesheet" />
+    <link href="/public/css/components/commerce-info.css" rel="stylesheet" />
+    <link href="/public/css/components/footer.css" rel="stylesheet" />
+    <link href="/public/css/utils.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header container">
+      <div class="header-content container-header">
+        <img class="header-logo" src="/public/assets/home-logo.png" alt="company logo" />
+        <div class="header-container">
+          <img class="header-logo-mobile" src="/public/assets/home-logo.png" alt="company logo" />
+          <nav class="nav-container">
+            <ul>
+              <li><a href="#home">ホーム</a></li>
+              <li><a href="#case-studies">制作事例</a></li>
+              <li><a href="#testimonials">お客様の声</a></li>
+              <li><a href="#faqs">よく頂く質問</a></li>
+            </ul>
+          </nav>
+          <button class="hamburger-btn">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor">
```

Comment:
(Not to need fix it)

[Nits]
If I were you I'd make this with HTML and CSS :)

Reply:
@sharray2918 
Per your recommendation I changed it to using HTML & CSS!